data/moves.js

```diff
@@ -13038,8 +13038,13 @@ exports.BattleMovedex = {
 		flags: {protect: 1, authentic: 1, mystery: 1},
 		onHit: function (target, source) {
 			if (source.template && (source.template.num === 493 || source.template.num === 773)) return false;
+			if (target.getTypes() === ['???']) return false;
```

Comment:
For the record, Slayer95 asked this because this code is completely wrong; `=== ['???']` will always be `false`.

Reply:
Yeah I got that eventually, I've just been working on getting everything right in another that I'll push to this when it's ready.